xarray/plot/facetgrid.py

```diff
@@ -714,23 +772,23 @@ def map(self, func, *args, **kwargs):
 
 
 def _easy_facetgrid(
-    data,
-    plotfunc,
-    kind,
-    x=None,
-    y=None,
-    row=None,
-    col=None,
-    col_wrap=None,
-    sharex=True,
-    sharey=True,
-    aspect=None,
-    size=None,
-    subplot_kws=None,
-    ax=None,
-    figsize=None,
-    **kwargs,
-):
+    data: DataArray,
+    plotfunc: Callable,
+    kind: Literal["line", "dataarray", "dataset"],
+    x: Hashable | None = None,
+    y: Hashable | None = None,
+    row: Hashable | None = None,
+    col: Hashable | None = None,
+    col_wrap: int | None = None,
```

Comment:
Isn't the "or None" here redundant? Or is this some style thing I'm unaware of?

Reply:
Do you mean that because as default it gets added anyway or because None is Hashable?
In any case I always prefer explicit instead of implicit.

Reply:
I meant the former, but fair enough, I was just wondering.